packages/devtools_app/lib/src/shared/object_tree.dart

```diff
@@ -0,0 +1,655 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
```

Comment:
The vast majority of this file is just a straight cut-paste from `debugger_model.dart`

Reply:
@bkonyi could you give some context around the changes to _createVariablesForAssociations?
![image](https://user-images.githubusercontent.com/1386322/171939600-f0cfd2b6-5e62-428f-887a-974dc15de049.png)